app/lib/actions/katello/repository/clone_docker_content.rb

```diff
@@ -2,14 +2,20 @@ module Actions
   module Katello
     module Repository
       class CloneDockerContent < Actions::Base
-        def plan(source_repo, target_repo)
+        def plan(source_repo, target_repo, filters)
+          filters = filters.docker unless filters.is_a? Array
+
+          if filters.any?
+            clause_gen = ::Katello::Util::DockerManifestClauseGenerator.new(source_repo, filters)
+            clause_gen.generate
+            copy_clauses = clause_gen.copy_clause
```

Comment:
I'll need to revisit this. Here is the code comment for rpm https://github.com/Katello/katello/blob/master/app/models/katello/content_view.rb#L584